src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNet/AspNetCoreMvc2Integration.cs

```diff
@@ -196,7 +196,7 @@ public static class AspNetCoreMvc2Integration
             TargetMaximumVersion = Major2)]
         public static void Rethrow(object context, int opCode, int mdToken)
         {
-            string methodDef = $"{ResourceInvoker}.{nameof(Rethrow)}({context?.GetType().FullName} context)";
+            string methodDef = $"{ResourceInvoker}.{nameof(Rethrow)}({context?.GetType().FullName + " "}context)";
```

Comment:
Was this change intentional? Seems like the result would be the same and the syntax was nicer before.

Reply:
I think you want something like:
```suggestion
            // only add the trailing space if the context is not null
            string contextTypeName = (context == null) ? "" : (context.GetType().FullName + " ");
            string methodDef = $"{ResourceInvoker}.{nameof(Rethrow)}({contextTypeName}context)";
```
Also, we can move this down into the `catch` where `methodDef` is used, so it's only created it needed.